dotcom-rendering/src/web/components/Snap.tsx

```diff
@@ -0,0 +1,32 @@
+import { css, Global } from '@emotion/react';
+
+const snapStyles = css`
+	overflow-y: hidden;
+	position: relative;
+`;
+
+type Props = {
+	enriched?: DCREnrichedContent;
+};
+
+export const Snap = ({ enriched }: Props) => {
+	return (
+		<div css={snapStyles}>
+			{enriched?.embedCss !== undefined && (
+				<Global styles={enriched?.embedCss} />
```

Comment:
Currently putting the styles for Snap in the page header along with the rest of the page styles. Should we copy frontend and put the styles for the snap next to the snap itself?

I don't feel too strongly either way, having them local might make it easier to debug.

Reply:
I would say we probably don't want to introduce global css on the page if it isn't _really_  global